modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/PartitionListener.java

```diff
@@ -205,15 +215,17 @@ private void handleUpdateCommand(UpdateCommand cmd, long commandIndex) {
     private void handleUpdateAllCommand(UpdateAllCommand cmd, long commandIndex) {
         storage.runConsistently(() -> {
             UUID txId = cmd.txId();
-            Map<RowId, BinaryRow> rowsToUpdate = cmd.getRowsToUpdate();
-            UUID commitTblId = cmd.getReplicationGroupId().getTableId();
-            int commitPartId = cmd.getReplicationGroupId().getPartId();
+            Map<UUID, ByteString> rowsToUpdate = cmd.rowsToUpdate();
+            UUID commitTblId = cmd.tablePartitionId().tableId();
+            int commitPartId = cmd.tablePartitionId().partitionId();
 
             if (!CollectionUtils.nullOrEmpty(rowsToUpdate)) {
-                for (Map.Entry<RowId, BinaryRow> entry : rowsToUpdate.entrySet()) {
-                    RowId rowId = entry.getKey();
-                    BinaryRow row = entry.getValue();
-
+                for (Map.Entry<UUID, ByteString> entry : rowsToUpdate.entrySet()) {
+                    UUID rowIdUuid = entry.getKey();
+                    RowId rowId = new RowId(partitionId, rowIdUuid.getMostSignificantBits(), rowIdUuid.getLeastSignificantBits());
```

Comment:
I think there is a public (int, UUID) constructor at this point

Reply:
Fixed.